src/Pages/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from '../components/Navbar/Navbar';
+import Books from '../components/Books/Books';
+import Categories from './Categories';
+
+const Home = () => (
+  <>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Books />} />
+      <Route path="/categories" element={<Categories />} />
+    </Routes>
```

Comment:
- [ ] Good job so far 🔥.  However, let's move everything concerning the route to the `index.js` file as it is not a good practice to have the routes in one of the components except `App` anyway. But it is a requirement to keep the route logic in the `index.js` file 😄.

Reply:
Yes, I had doubts about that. Now it is clear, I will do this change. Thank you 😃 👍